yb-voyager/src/queryparser/query_parser.go

```diff
@@ -41,6 +41,10 @@ func (qp *QueryParser) Parse() error {
 }
 
 func (qp *QueryParser) GetUnsupportedQueryConstructs() ([]utils.UnsupportedQueryConstruct, error) {
+	if qp.ParseTree == nil {
+		return nil, fmt.Errorf("query's parse tree is null for some reason, skipping unsupported construct check")
```

Comment:
Let's just say "query's parse tree is nil" ? 
Since it's an error, we don't have to say "skipping ... "